File: commonlib/src/main/java/com/app/commonlib/LogUtils.java
package com.app.commonlib;

import android.util.Log;

public class LogUtils {
    private static final String TAG = BuildConfig.LIBRARY_PACKAGE_NAME;

    public static void e(String content) {
        if (BuildConfig.DEBUG)
            Log.e(TAG, content);
    }

    public static void i(String content) {
        if (BuildConfig.DEBUG)
            Log.i(TAG, content);
    }

    public static void v(String content) {
        if (BuildConfig.DEBUG)
            Log.v(TAG, content);
    }

    public static void e(String tag, String content) {
        if (BuildConfig.DEBUG)
            Log.e(tag, content);
    }

    public static void i(String tag, String content) {
        if (BuildConfig.DEBUG)
            Log.i(tag, content);
    }

    public static void v(String tag, String content) {
        if (BuildConfig.DEBUG)
            Log.v(tag, content);
    }

}

File: commonlib/src/main/java/com/app/commonlib/utils/LogUtils.kt
package com.app.commonlib.utils

import android.util.Log
import com.app.commonlib.BuildConfig

/**
 * @ClassName DeviceUtils
 * @Author DYJ
 * @Date 2020/8/18 20:02
 * @Version 1.0
 * @Description 日志
 */
object LogUtils {

    private const val TAG = BuildConfig.LIBRARY_PACKAGE_NAME

    fun e(content: String) {
        if (BuildConfig.DEBUG) Log.e(TAG, content)
    }

    fun i(content: String) {
        if (BuildConfig.DEBUG) Log.i(TAG, content)
    }

    fun v(content: String) {
        if (BuildConfig.DEBUG) Log.v(TAG, content)
    }

    fun e(tag: String, content: String) {
        if (BuildConfig.DEBUG) Log.e(tag, content)
    }

    fun i(tag: String, content: String) {
        if (BuildConfig.DEBUG) Log.i(tag, content)
    }

    fun v(tag: String, content: String) {
        if (BuildConfig.DEBUG) Log.v(tag, content)
    }
}